rosetta-runtime/src/main/java/com/rosetta/model/lib/meta/Key.java

```diff
@@ -234,14 +231,14 @@ public Validator<? super Key> validator() {
 			return new Validator<Key>() {
```

Comment:
I think we should deprecate this one too, and rather create a `KeyValidator implements RosettaModelObjectValidator<Key>` using our new infrastructure.